app/src/main/java/org/jellyfin/androidtv/details/ItemListActivity.java

```diff
@@ -759,14 +753,13 @@ private void stopRotate() {
 
     protected void updateBackground(String url) {
         if (url == null) {
-            BackgroundManager.getInstance(this).setDrawable(mDefaultBackground);
+            BackgroundManager.getInstance(this).setDrawable(null);
```

Comment:
seems like a lot of duplicate code but I wouldn't want to consolidate it either haha

Reply:
Yeah. I briefly looked at moving this to the parent activity classes, but they are "borrowed" from AOSP so a larger refactor should probably happen at some point to remove the Apache licensed code.